modules/core/src/main/java/org/apache/ignite/internal/processors/cache/affinity/GridCacheAffinityImpl.java

```diff
@@ -218,20 +219,40 @@ public GridCacheAffinityImpl(GridCacheContext<K, V> cctx) {
         return res;
     }
 
-    /** {@inheritDoc} */
+    /** @deprecated Use mapKeyToPrimaryAndBackupsList(K) instead */
+    @Deprecated
     @Override public Collection<ClusterNode> mapKeyToPrimaryAndBackups(K key) {
         A.notNull(key, "key");
 
         return cctx.affinity().nodesByPartition(partition(key), topologyVersion());
     }
 
     /** {@inheritDoc} */
+    @Override
+    public List<ClusterNode> mapKeyToPrimaryAndBackupsList(K key) {
```

Comment:
Please, write it in one line: `@Override public`
Any reason to copy-paste implementation from `mapKeyToPrimaryAndBackupsList`?